main.go

```diff
@@ -26,6 +20,31 @@ func main() {
 	ContributeAndGetVoucher()
 }
 
+func WaitForSequencerToBeReady() {
+	spinnerChars := []string{"🌕", "🌖", "🌗", "🌘", "🌑", "🌒", "🌓", "🌔"}
+	spinnerIndex := 0
+	attempts := 0
+	removeLine := "\u001B[A\u001B[2K"
+	state := GetSequencerState()
+	for state != SEQUENCER_ACCEPTING {
+		message := "Sequencer currently not accepting new contributions, waiting..."
+		status := fmt.Sprintf("[Attempt %d - Last Checked: %s]", attempts, time.Now().String())
+
+		fmt.Printf("\r%s", removeLine)
+		fmt.Printf("%s\n", message+spinnerChars[spinnerIndex])
+		fmt.Printf("  |- %s", status)
+
+		spinnerIndex = (spinnerIndex + 1) % len(spinnerChars)
+		attempts += 1
+
+		time.Sleep(1 * time.Second)
```

Comment:
Let's not thrash the sequencer or at least make it configurable – some people are running a lot of these

```suggestion
		time.Sleep(5 * time.Second)
```